.github/workflows/centos.yml

```diff
@@ -0,0 +1,13 @@
+name: Centos
+on:
```

Comment:
Let it run in every push because it also checks the server package and it just takes a couple of minutes.